src/allotropy/parsers/unchained_labs_lunatic/unchained_labs_lunatic_structure.py

```diff
@@ -63,40 +64,64 @@ def _create_measurement(
         )
 
     measurement_identifier = random_uuid_str()
+    error_documents: list[ErrorDocument] = []
     calculated_data.extend(
-        _get_calculated_data(well_plate_data, wavelength_column, measurement_identifier)
+        _get_calculated_data(
+            well_plate_data, wavelength_column, measurement_identifier, error_documents
+        )
     )
+    absorbance = well_plate_data.get(float, wavelength_column)
+
+    if absorbance is None:
+        error_documents.append(
+            ErrorDocument(
+                error=NOT_APPLICABLE,
+                error_feature=DETECTION_TYPE.lower(),
+            )
+        )
     return Measurement(
         type_=MeasurementType.ULTRAVIOLET_ABSORBANCE,
         device_type=DEVICE_TYPE,
         detection_type=DETECTION_TYPE,
         identifier=measurement_identifier,
         detector_wavelength_setting=float(wavelength),
         electronic_absorbance_reference_wavelength_setting=background_wavelength,
-        electronic_absorbance_reference_absorbance=background_absorbance,
-        absorbance=well_plate_data.get(float, wavelength_column, NaN),
+        measurement_custom_info={
+            "electronic_absorbance_reference_absorbance": background_absorbance
+        },
+        absorbance=absorbance if absorbance is not None else NEGATIVE_ZERO,
         sample_identifier=well_plate_data[str, "Sample name"],
         location_identifier=well_plate_data[str, "Plate Position"],
         well_plate_identifier=well_plate_data.get(str, "Plate ID"),
         batch_identifier=well_plate_data.get(str, "Sample Group"),
         firmware_version=header.get(str, "Client version"),
-        path_length=float(path_length) if path_length else None,
+        sample_custom_info={
+            "path length": path_length if path_length else None,
```

Comment:
Let's still cast this to a float

```suggestion
            "path length": float(path_length) if path_length is not None else None,
```